eng/pipelines/templates/steps/deploy-resources.yml

```diff
@@ -0,0 +1,20 @@
+parameters:
+  SecretSuffix: ''
+  ServiceDirectory: not-set
+  AdditionalParameters: ''
+  Location: 'westus2'
+  Environment: ''
+
+steps:
+  - template: ../common/TestResources/deploy-test-resources.yml
+    paramters:
+      ServiceDirectory: ${{ parameters.ServiceDirectory }}
```

Comment:
While the templating helped reduce the duplication I'm still curious if all these secrets are necessary, given we are creating the resources dynamically.

Reply:
We chatted last week about these secrets. These have been consolidated as much as can be done.